docs/release.md

```diff
@@ -40,9 +40,11 @@ git clone git@github.com:rancher/turtles.git
 ```bash
 # Create tags locally
 git tag -s -a ${RELEASE_TAG} -m ${RELEASE_TAG}
+git tag test/${RELEASE_TAG}
```

Comment:
Maybe we can sign tests tag as well? Same below, line 47
```suggestion
git tag -s  -a test/${RELEASE_TAG} -m “Testing framework ${RELEASE_TAG}”
```

Reply:
Yes, let's sign and annotate it